### ISSUE TITLE
Android App Designing

### ISSUE THREAD

**Issue by arpitgupta2908** [Innovation-Cell/Issue-Tracking#23]

**Comment by kishore19a** [comment]
to be floated in the institute.....talked to some people who are ready to design but not develop the app.

**Comment by kishore19a** [comment]
car on/off module to be completed by this sunday
Ganesh and Naman have started working on it.

**Comment by mistryrakesh** [comment]
Start/Stop module finished and tested. We can ahead with advanced functionality now.

**Comment by kishore19a** [comment]
Port 80 issue resolved??

**Comment by mistryrakesh** [comment]
No, I had to use a **VPN** to get it working. But it should work using a dongle such as that of vodafone or airtel since ports are not blocked in them.
